test/src/main/java/org/apache/accumulo/test/metrics/MetricsIT.java

```diff
@@ -102,6 +102,7 @@ public void confirmMetricsPublished() throws Exception {
     // @formatter:off
     Set<String> unexpectedMetrics =
             Set.of(METRICS_COMPACTOR_MAJC_STUCK,
+                    METRICS_COMPACTOR_BUSY,
```

Comment:
This may need to move from the unexpected metrics to the flaky metrics when merging to elasticity.